zebrad/tests/acceptance.rs

```diff
@@ -82,12 +82,22 @@ fn default_test_config() -> Result<ZebradConfig> {
         ..zebra_consensus::Config::default()
     };
 
+    let force_use_color = !matches!(
+        env::var("ZEBRA_FORCE_USE_COLOR"),
+        Err(env::VarError::NotPresent)
+    );
```

Comment:
Can we also use this env var in `zebra_test::init()`?
That way, I can get coloured output even when I'm writing unit test logs to a file.

The colour is set by a filter layer:
https://docs.rs/tracing-subscriber/0.3.8/tracing_subscriber/?search=with_ansi

(I don't know what the current unit test behaviour is, but we can just use this env var to switch colour on and off.)

Reply:
Unit tests are already colored by default :tada: (see: https://docs.rs/tracing-subscriber/latest/src/tracing_subscriber/fmt/fmt_layer.rs.html#582)

We only need to force color for the spawned Zebra processes, because Zebra tries to disable color if it doesn't detect a terminal.